scripts/create_istio_system.sh

```diff
@@ -38,17 +38,17 @@ else
   fi
 fi
 
-if kubectl get namespace cattle-dashboards; then
+if kubectl get namespace cattle-dashboards --kubeconfig=$HOME/sa.kubeconfig; then
   # delete existing managed istio dashboards
-  kubectl delete configmap -n cattle-dashboards -l istio_dashboard=1
+  kubectl delete configmap -n cattle-dashboards -l istio_dashboard=1 --kubeconfig=$HOME/sa.kubeconfig
   for dashboard in /usr/local/app/dashboards/*.json; do
     name="istio-dashboard-$(basename ${dashboard} ".json")"
     # replace datasource input placeholder in dashboard with datasource name in rancher-monitoring
     sed -i 's/${DS_PROMETHEUS}/Prometheus/g' ${dashboard}
     # if a configmap with the name already exists and is not managed by this installer script, skip it and do not add the istio_dashboard label
-    if kubectl create configmap ${name} -n cattle-dashboards --from-file=$(basename ${dashboard})=${dashboard}; then
-      kubectl label configmap ${name} -n cattle-dashboards grafana_dashboard=1
-      kubectl label configmap ${name} -n cattle-dashboards istio_dashboard=1
+    if kubectl create configmap ${name} -n cattle-dashboards --from-file=$(basename ${dashboard})=${dashboard} --kubeconfig=$HOME/sa.kubeconfig; then
+      kubectl label configmap ${name} -n cattle-dashboards grafana_dashboard=1 --kubeconfig=$HOME/sa.kubeconfig
+      kubectl label configmap ${name} -n cattle-dashboards istio_dashboard=1 --kubeconfig=$HOME/sa.kubeconfig
```

Comment:
If you're not using another kubeconfig at `KUBECONFIG` env var, you can set it to `--kubeconfig=$HOME/sa.kubeconfig` so it doesn't need to be set each time.